.lando.yml

```
@@ -39,7 +39,15 @@ tooling:
   # Provide a command to install Drupal.
   install:
     service: appserver
-    cmd: /app/vendor/bin/drush --root=/app/web site:install --account-mail=noreply@email.arizona.edu --account-name=azadmin --account-pass=azadmin --db-url=mysql://drupal9:drupal9@database:3306/drupal9 -y --verbose && /app/vendor/bin/drush en -y azqs_module_project
+    cmd:
+      - /app/vendor/bin/drush --root=/app/web site:install
+        --account-mail=noreply@email.arizona.edu --account-name=azadmin
+        --account-pass=azadmin
+        --db-url=mysql://drupal9:drupal9@database:3306/drupal9 -y --verbose
+      - /app/vendor/bin/drush en -y azqs_module_project
```

Minor consistency nitpick:
```suggestion
      - /app/vendor/bin/drush --root=/app/web pm:install -y azqs_module_project
```